src/database.js

```diff
@@ -674,6 +680,23 @@ Database.prototype.getTransaction = function(options, callback) {
  * });
  *
  * //-
+ * // Alternatively, set `query.json` to `true`, and this step will be performed
+ * // automaticaly.
+ * //-
+ * database.run(query, function(err, rows) {
+ *   if (err) {
+ *     // Error handling omitted.
+ *   }
+ *
+ *   const row1 = rows[0];
```

Comment:
looks really weird even for a sample code, can we call it `row0`, or `firstRow`, or just `row` ? :)